docs/cube/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _cube:
+
+************************************
+Cube Style Analysis (`gammapy.cube`)
+************************************
+
+.. currentmodule:: gammapy.cube
+
```

Comment:
I meant to say: add a link to those presentations instead of putting a long description here.